tools/benchcomp/configs/perf-regression.yaml

```diff
@@ -10,13 +10,13 @@ variants:
   kani_new:
     config:
       directory: new
-      command_line: scripts/kani-perf.sh
+      command_line: "scripts/setup/ubuntu/install_deps.sh && scripts/kani-perf.sh"
```

Comment:
Should we just run `install_cbmc.sh` instead? The `bench` workflow runs `install_deps.sh` as part of the `setup` action.

Reply:
Possibly - I guess it is the only dependency that could affect performance?

Reply:
We also install kissat as part of this, and a PR changing kissat versions could also change performance characteristics. So I'd rather not make this proposed change.